### DISCUSSION TITLE
Have an event with a base price + additional seats. Is that possible?

### DISCUSSION THREAD

**Discussion by Konafets** [pretix/pretix#3393] [Q&A] [answered]
I have a customer request and I am not sure if this is possible with Pretix. The customer offers an event on a fixed date. The price of the event is 300 € and includes three persons. That would be the "base price", if someone books that event. Additionally he can add more persons (6 more) to that event but with every additional person the price goes up (100 / person).

How can I implement this in Pretix?

**Comment by raphaelm** [comment, marked as answer]
There's multiple ways to do this, either via the "discount" feature or through "add-on products". See also

https://docs.pretix.eu/en/latest/user/events/guides/packages.html
https://docs.pretix.eu/en/latest/user/events/guides/groups.html

**Reply by Konafets** [reply]
Thank for the links. I am a bit confused by "discount" in this regard. But I think I got it working with the add-on.
